src/containers/main-page/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Box } from '@mui/material';
+import { FormWrapperStyled, MainPageContainerStyled, PreviewWrapperStyled } from './styled';
+import CVForm from './cv-form';
+import CvFormStepperProvider from './cv-form/state/CVFormStepperProvider';
+
+const MainPage = function (): JSX.Element {
+  return (
+    <MainPageContainerStyled container>
+      <FormWrapperStyled
+        item
+        xs={12}
+        sm={12}
+        md={12}
+        lg={12}
+        xl={6}
+      >
+        <CvFormStepperProvider>
+          <CVForm />
+        </CvFormStepperProvider>
+      </FormWrapperStyled>
+      <PreviewWrapperStyled
+        item
+        xs={12}
+        sm={12}
+        md={12}
+        lg={12}
+        xl={6}
+      >
+        <Box>
+          Preview
```

Comment:
Please place to the constants this text.

Reply:
It's mockup text that will be removed today. Next time I'll add // TODO:, promise :)